### ISSUE TITLE
pyarrow.lib.ArrowNotImplementedError: Unhandled type for Arrow to Parquet schema conversion: halffloat

### ISSUE THREAD

**Issue by kopyl** [apache/arrow#39790]
### Describe the bug, including details regarding any error messages, version, and platform.

Issue description: https://github.com/huggingface/datasets/issues/6595

And why the hell this issue was closed without fixing it first???
https://github.com/apache/arrow/issues/2691

### Component(s)

Parquet, Python

**Comment by assignUser** [comment]
The issue was closed because at the time we were using jira to track issues, which is pretty clear when you look at the issue. Please try to be respectful when asking for help.

The just released pyarrow 15.0.0 supports float16 thanks to year long work from multiple contributors, thanks for appreciating it. 
```
import pyarrow as pa
import pyarrow.parquet as pq
import numpy as np
arr = pa.array(np.float16([0.1, 2.2, 3]))
table = pa.table({'a': arr})
pq.write_table(table, "test_halffloat.parquet") 
```

**Comment by kopyl** [comment]
@assignUser thank you very much, sorry for being rude
